### PR TITLE
Replaces gun spawners on eris with makeshift/handmade gun spawners, unblacklist handmade armor from spawn

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request
You thought I was going to stop?
Instead of just blacklisting all but handmade guns, I add and change the spawners on Eris.dmm, so there should still be guns in junk beam and deepmaint, as well as OS dungeons
## Why It's Good For The Game
I still was ~~asked~~ forced to do this by Rosalimo and FortuneMaster
no more finding .40 miller in a trash pile roundstart, you'll have to at least go junk beaconing for that now. Or use gun assemblies. Or go to guild. Or do literally anything else but junking please-
~~use the goddamn melee weaponry~~
## Changelog
:cl:
add: handmade guns spawner
balance: replace all gun spawners on Eris.dmm with handmade gun spawners
tweak: unblacklisted handmade armor from spawning
/:cl:

### CODE REVIEW

**Comment by zombiemimic** [comment]
Much better

File: code/modules/scrap/scrap.dm [hunk 1]
```diff
@@ -429,7 +429,7 @@ GLOBAL_LIST_EMPTY(scrap_base_cache)
 	loot_max = 10
 	rarity_value = 90
 	loot_tags = list(
-		SPAWN_GUN = 0.3,
+		SPAWN_GUN_HANDMADE = 0.3,
 		SPAWN_PART_GUN = 3,
```
Comment (Humonitarian) [inline code/modules/scrap/scrap.dm:433]: Consider decreasing SPAWN_GUN_HANDMADE to ~0.1-0.2, considering the lack of variety that spawner provides.

Comment (Kegdo) [reply code/modules/scrap/scrap.dm:433]: I think it's fine as is, the 0.3 is really rare already.

**Comment by Kegdo** [comment]
This one DOES NOT need testing, I used spawn templates on localhost and it spawned actual guns, so all gud

File: code/modules/clothing/spacesuits/rig/rig.dm [hunk 2]
```diff
@@ -23,7 +23,7 @@
 	item_flags = DRAG_AND_DROP_UNEQUIP|EQUIP_SOUNDS
 	spawn_tags = SPAWN_TAG_RIG
 	rarity_value = 10
-	bad_type = /obj/item/rig
+	bad_type = /obj/item/rig //TODO: Resprite these, remove old bay leftover RIGs, add a RIG to moeb R&D, add more RIGs in general
```
Comment (Humonitarian) [inline code/modules/clothing/spacesuits/rig/rig.dm:26]: What's this comment doing out here?

Comment (Kegdo) [reply code/modules/clothing/spacesuits/rig/rig.dm:26]: it's a TODO :)

File: code/game/objects/random/biomes/guns.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /obj/landmark/loot_biomes/obj/guns
 	name = "Biome: Guns"
-	main_tags = list(SPAWN_GUN = 2, SPAWN_GUN_ENERGY, SPAWN_GUN_SHOTGUN)
+	main_tags = list(SPAWN_GUN_HANDMADE)
```
Comment (Humonitarian) [inline code/game/objects/random/biomes/guns.dm:3]: Correct me if I'm wrong, but aren't biomes mainly used in deepmaint to determine the amount of traps and mobs?
To quote evankhell,

> biome spawners: These are a smart spawner that prioritizes the most dangerous mobs and traps depending on the amount of loot in the area.

Comment (Kegdo) [reply code/game/objects/random/biomes/guns.dm:3]: I think it affects everything, both Eris and deepmaint and stuff. Eh, if anything, I'll make a hotfix; bringing back the old main tags to this file.

**Comment by TheShown911** [comment]
This does need testing.

File: code/game/objects/random/biomes/guns.dm (hunk 3 above)
Comment (MLGTASTICa) [reply code/game/objects/random/biomes/guns.dm]: preety sure they're use for junk beacon

Comment (Humonitarian) [reply code/game/objects/random/biomes/guns.dm]: > preety sure they're use for junk beacon

It has children to spawn roaches and spiders, which alongside Evankhell's comment is why I assume this is used in deepmaint.

**Comment by ValoTheValo** [comment]
he still blacklisted armor spawns epic troll

**Comment by SugarBone** [comment]
As a true Eris player, I do not condone this PR and, in fact, condemn it. This should not be merged.

**Comment by Alliostra** [comment]
why would you make junk beacon still contain guns while regular maint piles don't? junk beacon is 0 risk, while in maint there's at least some chance to get attacked by roaches or antags.

**Comment by TheShown911** [comment]
> why would you make junk beacon still contain guns while regular maint piles don't? junk beacon is 0 risk, while in maint there's at least some chance to get attacked by roaches or antags.

Play game, JB got reworked

**Comment by Humonitarian** [comment]
> why would you make junk beacon still contain guns while regular maint piles don't? junk beacon is 0 risk, while in maint there's at least some chance to get attacked by roaches or antags.

Carp moment

**Comment by drexample** [comment]
Requires to be ran again on test server with the gun assemblies

**Comment by Kegdo** [comment]
i am visibly upset

**Comment by SirRichardFrancis** [comment]
> i am visibly upset

I converted spaces to tabs in lutty.dm, just redo your changes there.

**Comment by Kegdo** [comment]
i am no longer visibly upset

**Comment by ValoTheValo** [comment]
This PR and its consequences have been a disaster for the spaceman race

**Pull request merged**